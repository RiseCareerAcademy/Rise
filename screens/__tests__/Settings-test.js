import 'react-native';
import React from 'react';
import settings from '../Settings';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const props = {
    };
  const tree = renderer.create(<settings {...props}>Snapshot test!</settings>).toJSON();

  expect(tree).toMatchSnapshot();
});
