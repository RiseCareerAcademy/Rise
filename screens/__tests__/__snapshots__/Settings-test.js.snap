// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "paddingTop": 23,
    }
  }
>
  <TextInput
    allowFontScaling={true}
    autoCapitalize="none"
    onChangeText={[Function]}
    placeholder="Change Email"
    placeholderTextColor="#000000"
    style={
      Object {
        "borderColor": "#000000",
        "borderWidth": 1,
        "height": 40,
        "margin": 15,
      }
    }
    underlineColorAndroid="transparent"
  />
  <TextInput
    allowFontScaling={true}
    autoCapitalize="none"
    onChangeText={[Function]}
    placeholder="Change Password"
    placeholderTextColor="#000000"
    style={
      Object {
        "borderColor": "#000000",
        "borderWidth": 1,
        "height": 40,
        "margin": 15,
      }
    }
    underlineColorAndroid="transparent"
  />
  <TextInput
    allowFontScaling={true}
    autoCapitalize="none"
    onChangeText={[Function]}
    placeholder="Confirm Password Change"
    placeholderTextColor="#000000"
    style={
      Object {
        "borderColor": "#000000",
        "borderWidth": 1,
        "height": 40,
        "margin": 15,
      }
    }
    underlineColorAndroid="transparent"
  />
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#000000",
        "height": 40,
        "margin": 15,
        "opacity": 1,
        "padding": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "white",
        }
      }
    >
       Save 
    </Text>
  </View>
</View>
`;
